server/src/fat/java/com/ibm/ws/lars/rest/ApiTest.java

```diff
@@ -1146,11 +1209,16 @@ public void test500Mapping() throws InvalidJsonAssetException, ParseException, I
     @Test
     public void testAssetReviews() throws InvalidJsonAssetException, IOException {
         Asset a = addLittleAsset("weather", "hot", "ground", "flat", "description", "lovely");
-        List<?> ars = repository.getAssetReviews(a.get_id());
+        List<Map<String, Object>> ars = repository.getAssetReviews(a.get_id());
         // Story 165844, for now assetreviews just returns an empty JSON array
-        assertEquals(
-                     "Should be an empty List",
-                     0, ars.size());
+        assertEquals("Should be an empty List", 0, ars.size());
+        RepositoryContext userRepo = RepositoryContext.toUserContext(repository);
+        try {
```

Comment:
While I can see your point, I'd also prefer it if we could avoid
* catching `AssertionError` - this causes confusion because it means that a call to `assertXyz()` does not actually cause the test to fail
* matching error message strings - as they are meant to be descriptive and may change in the future